elytron/src/main/java/org/wildfly/extension/elytron/ElytronSubsystemTransformers.java

```diff
@@ -88,6 +88,7 @@ private static void from4(ChainedTransformationDescriptionBuilder chainedBuilder
                         ))
                 ), PRINCIPAL_QUERIES)
                 .end();
+        builder.rejectChildResource(PathElement.pathElement(ElytronDescriptionConstants.MAPPED_ROLE_MAPPER));
```

Comment:
please cover this by transformers tests too (check https://hkalina.github.io/2018/05/15/transformers/ )